### PR TITLE
docs: simplify termux installation process

### PR DESCRIPTION
**Pull request by thx2001r**
1) Updated information to use PRoot (simpler than Andronix and the way supported by Termux) to create and access the Debian distro.
2) Added helpful information on using PRoot with your dev environment.
3) Cleaned up spelling, grammar, and made documentation more consistent between sections.

Fixes #

### CODE REVIEW

**Comment by thx2001r** [comment]
Please take a look at the screen recording demonstrating the updated documentation working on the latest Termux on Android 12 on a Samsung Galaxy S21 in DeX mode:

https://youtu.be/Stu8eSALLw8

File: docs/termux.md [hunk 1]
```diff
@@ -10,40 +10,44 @@
...
 
 ## Install
 
 1. Get [Termux](https://f-droid.org/en/packages/com.termux/) from **F-Droid**.
-2. Install Debian by running the following.
+2. Install Debian by running the following:
    - Run `termux-setup-storage` to allow storage access, or else code-server won't be able to read from `/sdcard`.\
-      If you used the Andronix command then you may have to edit the `start-debian.sh` script to mount `/sdcard` just as simple as uncommenting the `command+=" -b /sdcard"` line.
-     > The following command was extracted from [Andronix](https://andronix.app/) you can also use [proot-distro](https://github.com/termux/proot-distro).
+     > The following command is from [proot-distro](https://github.com/termux/proot-distro) you can also use [Andronix](https://andronix.app/).
```
Comment (jsjoeio) [inline docs/termux.md:22]:
```suggestion
     > The following command is from [proot-distro](https://github.com/termux/proot-distro), but you can also use [Andronix](https://andronix.app/).
```

File: docs/termux.md [hunk 2]
```diff
@@ -10,40 +10,44 @@
...
      > After Debian is installed the `~ $` will change to `root@localhost`.
 
 ```bash
-pkg update -y && pkg install wget curl proot tar -y && wget https://raw.githubusercontent.com/AndronixApp/AndronixOrigin/master/Installer/Debian/debian.sh -O debian.sh && chmod +x debian.sh && bash debian.sh
+pkg up -y && pkg in proot-distro -y && proot-distro install debian && proot-distro login debian
```
Comment (jsjoeio) [inline docs/termux.md:26]: maybe a silly question but is the `in ` supposed to be there? (I don't use `pkg` much)

File: docs/termux.md [hunk 3]
```diff
@@ -82,11 +86,11 @@ Potential Workaround :
...
 
-1. Create a new user by running `useradd username -m`.
-2. Change the password by running `passwd username`.
-3. Give your new user sudo access by runnning `visudo`, scroll down to `User privilege specification` and add the following line after root `username ALL=(ALL:ALL) ALL`.
-4. Now edit the `/etc/passwd` file with your commadline editor of choice and at the end of the line that specifies your user change `/bin/sh` to `/bin/bash`.
-5. Now switch users, by running `su - username`
+1. Create a new user by running `useradd <username> -m`.
+2. Change the password by running `passwd <username>`.
+3. Give your new user sudo access by running `visudo`, scroll down to `User privilege specification` and add the following line after root `username ALL=(ALL:ALL) ALL`.
+4. Now edit the `/etc/passwd` file with your commandline editor of choice and at the end of the line that specifies your user change `/bin/sh` to `/bin/bash`.
```
Comment (jsjoeio) [inline docs/termux.md:92]:
```suggestion
4. Now edit the `/etc/passwd` file with your command line editor of choice and at the end of the line that specifies your user change `/bin/sh` to `/bin/bash`.
```

File: docs/termux.md [hunk 4]
```diff
@@ -82,11 +86,11 @@ Potential Workaround :
...
+5. Now switch users, by running `su - <username>`
```
Comment (jsjoeio) [inline docs/termux.md:93]:
```suggestion
5. Now switch users by running `su - <username>`
```

**Review by jsjoeio — CHANGES_REQUESTED** [review]
This is a great improvement with all the extra info added! Thank you so much!

Just a couple minor suggestions

File: docs/termux.md (hunk 2 above)
Comment (thx2001r) [reply docs/termux.md:26]: Oh fair.  pkg in and up are shorthand for install and update.

Comment (thx2001r) [reply docs/termux.md]: Updated in a subsequent commit.  Thanks!

**Review by jsjoeio — APPROVED** [review]
[the force is strong with this one](https://www.youtube.com/watch?v=Iyr74Rs6BWU)

**Comment by jsjoeio** [comment]
My least favorite part of our build step. Can you run `yarn fmt`? Sorry for the hassle!

image

**Comment by thx2001r** [comment]
Not sure how to do that as I made the commits and pull request directly in Github and don't have a code-server dev environment setup.  Ideas?

**Comment by jsjoeio** [comment]
> Not sure how to do that as I made the commits and pull request directly in Github and don't have a code-server dev environment setup. Ideas?

Hmm, what if I submit a PR to your fork, you merge and then update this branch?

**Comment by thx2001r** [comment]
Ok, was able to run yarn fmt and commited the change that way.  Does that work?

**Comment by jsjoeio** [comment]
> Ok, checked out my branch locally, and was able to npm run fmt and commited the change that way. Does that work?
It should 🤞🏼

**Comment by thx2001r** [comment]
Cool, looks like it worked!  Cloned the branch to local on Termux and checked in with code-server running on the same Termux for fun.  What is the next step?

**Pull request merged**